internal/install/static_snapshot_yml.go

```diff
@@ -84,9 +84,9 @@ services:
       kibana:
         condition: service_healthy
     healthcheck:
-      test: "elastic-agent status"
-      retries: 60
-      interval: 1s
+      test: "curl -f http://127.0.0.1:8220/api/status | grep HEALTHY 2>&1 >/dev/null"
+      retries: 12
+      interval: 5s
```

Comment:
Just curious, is`/api/status` stable? If so, then I'm wondering why we switched to `elastic-agent status` in the first place?

Reply:
There was a belief that `elastic-agent status` is sufficient, but it seems that it doesn't. Quote from @blakerouse :

> probably need to add some type of stable checking to ensure that the status reported is stable and not always changing

Reply:
Yes, I'm aware of this. This is the reason why checks are not so frequent (every 5s instead of standard 1s).